### PR TITLE
New time system - system_time, duration

### PR DESCRIPTION
**Pull request by namse**
~Instant: Time since namui init.~
System time: literally system time. = `what's the time?`'s answer
Duration: The same concept of previous `Time`. I renamed it to match with std's one.

Use case
```rust
let time = namui::time::since_start();
let x = (time.as_secs_f32() * 10.0).round() % 40.0;
Xy::new(x.px(), 0.0.px())
```

### CODE REVIEW

**Comment by namse** [comment]
@bigfoodK Check before review whether the functionality is enough or not

**Comment by namse** [comment]
I'm gonna fix something. I'm planning to remove Instant struct.

**Comment by namse** [comment]
Hmm. I though removing Instant is good idea, but that made a confusion between the `Timestamp` and `Duration`.
The last commit I removed `Instant` and made `time::since_start() -> Duration`. But I need `Timestamp` to set when the action is started.
Maybe I need to revive `Instant` or make `Timestamp` or `Instant` type.

**Review by bigfoodK — APPROVED** [review]
The sample work for me

File: namui/namui-type/src/types/units/time.rs-delete-this
[the remark this answers is not in the document]
Comment (namse) [reply namui/namui-type/src/types/units/time.rs-delete-this:1]: yes

**Pull request merged**